docs/charts.md

```diff
@@ -45,6 +45,7 @@ The `Chart.yaml` file is required for a chart. It contains the following fields:
 ```yaml
 name: The name of the chart (required)
 version: A SemVer 2 version (required)
+kubeVersion: A Kubernetes version the cluster needs to match (optional)
```

Comment:
Can we also describe that it is a semver here? Something like:

"Specifies the compatible version(s) of Kubernetes as a SemVer version (optional)"

Reply:
I’d change that to a “semver range”, as it’s specifying a range of known kubernetes versions this chart is compatible with.

Reply:
I adjusted the wording to your comments